nep-xx.mediawiki

```diff
@@ -0,0 +1,103 @@
+|                                                              |      | "name": "update",                                            |
+|                                                              |      | "safe": false,                                               |
+|                                                              |      | "parameters": [                                              |
+|                                                              |      | {                                                            |
+|                                                              |      | "name": "nefFile",                                           |
+|                                                              |      | "type": "ByteArray"                                          |
+|                                                              |      | },                                                           |
+|                                                              |      | {                                                            |
+|                                                              |      | "name": "manifest",                                          |
+|                                                              |      | "type": "String"                                             |
+|                                                              |      | },                                                           |
+|                                                              |      | {                                                            |
+|                                                              |      | "name": "data",                                              |
+|                                                              |      | "type": "Any"                                                |
+|                                                              |      | }                                                            |
+|                                                              |      | ],                                                           |
+|                                                              |      | "returntype": "Void"                                         |
+|                                                              |      | }                                                            |
+|                                                              |      | </pre>                                                       |
+|                                                              |      |                                                              |
+|                                                              |      | Updating a smart contract MUST have <code>nefFile</code> and <code>manifest.json</code>. |
```

Comment:
Why suggest the requirement that both manifest and NEF must be present? The management contract requires one of the two or both.
https://github.com/neo-project/neo/blob/5f4c14e3801c691ceccb5705c634e9e55dd3bba1/src/Neo/SmartContract/Native/ContractManagement.cs#L233

Reply:
Done